src/main/kotlin/no/skatteetaten/aurora/boober/service/AuroraDeploymentSpecValidator.kt

```diff
@@ -153,6 +154,13 @@ class AuroraDeploymentSpecValidator(
         }
     }
 
+    fun validateSecretNames(deploymentSpecInternal: AuroraDeploymentSpecInternal) {
+        deploymentSpecInternal.volume?.secrets?.forEach { secret ->
+            if (secret.name.length > 63) {
+                throw AuroraDeploymentSpecValidationException("The name of the secretVault=${secret.name} is too long. Max 63 characters. Note that we prefix the vaultName with @name@- it does not already")
```

Comment:
`Note that we prefix the vaultName with @name@- it does not already`

Skal det være noe sånt?
`Note that we prefix the vaultName with @name@- if it does not already contain it`

Reply:
`Note that we ensure that the name starts with @name@-`

Skrev jeg nå.